chapters/002-smol-internet/README.md

```diff
@@ -190,251 +149,205 @@ Let's go back to our `docker-compose.yml` and give `boudi` an additional network
+Now back to our regularly scheduled exploration!
+
+### Can `tara` ping `boudi` on the `squasheeba` network?
 
-But wait! That's not the behavior we want... Let's get rid of that default gateway. To do it manually, we can `hopon tara` and run
+We've already seen that `tara` can respond to `boudi`'s `ping`s that were issued from `boudi`'s interface on `doggonet`. We can double check that `tara` can initiate the `ping`, just for fun:
 
 ```bash
-root@tara:/# ip route del default
+root@tara:/# ping 10.1.2.3 -c 2
+PING 10.1.2.3 (10.1.2.3) 56(84) bytes of data.
+64 bytes from 10.1.2.3: icmp_seq=1 ttl=64 time=0.245 ms
+64 bytes from 10.1.2.3: icmp_seq=2 ttl=64 time=0.242 ms
+
+--- 10.1.2.3 ping statistics ---
+2 packets transmitted, 2 received, 0% packet loss, time 1041ms
+rtt min/avg/max/mdev = 0.242/0.243/0.245/0.001 ms
 ```
 
-Now, when we try to ping `boudi` on the `squasheeba` network, we get the failure we expect:
+Now... let's see if `tara` is able to ping `boudi`'s interface on the `squasheeba` network...
 
 ```bash
 root@tara:/# ping 10.1.1.3
 ping: connect: Network is unreachable
 ```
 
-And, watching carefully, we see that the `tcpdump` on `tara` no longer has any output.
+Lovely! This is because, while `boudi` has an interface on the `doggonet` network and `tara` and `boudi` can directly communicate on that network, `tara` doesn't know anything about the `squasheeba` network; not even that `boudi` knows about it. We need to define a route for `tara` that tells it how to get to the `squasheeba` network!
 
-BOOM! Good job, team.
+### Make `tara` ping hosts on the `squasheeba` network
 
-For the future, we can make this the default setup by changing the  [start-up script](./init/start-up.sh) to look like:
+The first thing we need to do is add a route from `tara` to the `squasheeba` network via `boudi`. Because `boudi` has routes to both `doggonet` and `squasheeba`, `boudi` can act as the gateway between the two. We can manage routes on our machines using the `ip route` command:
 
 ```bash
-#!/bin/sh
-
-/usr/sbin/ip route delete default
-
-/bin/sleep infinity
+root@tara:/# ip route add 10.1.1.0/24 via 10.1.2.3
 ```
 
-This will delete the default route when the docker container boots up.
-
-### Make `tara` ping hosts on the `squasheeba` network
-
-So let's see if we can get `tara` to ping `boudi`, or `pippin` for that matter, on the `squasheeba` network without using the default gateway router. The first thing we need to do is add a route from `tara` to the `squasheeba` network via `boudi`. Because `boudi` has routes to both `doggonet` and `squasheeba`, `boudi` can act as the gateway between the two.
+This command defines the network, `10.1.1.0/24` and then says that routes to that network should use a machine that exists on a network it has an interface on, namely, `boudi` at `10.1.2.3`. Now, if we check the routes that `tara` knows about, we'll see the route defined in `tara`'s routing table:
```

Comment:
```suggestion
This command identifies the network, `10.1.1.0/24` (a.k.a. "squasheeba") and then says: "Any time you have a packet for this network, you should send it to `10.1.2.3` (a.k.a. `boudi`), cuz that dude knows all about it." 

Now, if we check the routes that `tara` knows about, we'll see the route defined in `tara`'s routing table:
```

`and then says that routes to that network should use a machine that exists on a network it has an interface on` felt a little awkward to me

Reply:
YES! Thank you! I had the same thought! Brilliant cleanup here!